src/python/director/screengrabberpanel.py

```diff
@@ -27,9 +27,11 @@ def __init__(self, widgets):
 
 
 class ScreenGrabberPanel(object):
+    # HACK
+    instance = None
 
     def __init__(self, view):
-
+        ScreenGrabberPanel.instance = self
```

Comment:
Sounds good to me. I'll rename the PR to WIP, to indicate that it'll change significantly.
Is it OK if we tackle this possible two weeks from now?